Lib/test/test_fractions.py

```diff
@@ -91,6 +92,170 @@ class DummyFraction(fractions.Fraction):
 def _components(r):
     return (r.numerator, r.denominator)
 
+def is_eq(a, b):
+    return type(a) == type(b) and (a == b or math.isclose(a, b))
+
+class One:
+    def __mul__(self, other):
```

Comment:
It would be great to have a docstring indicating the intent here - it's not obvious what the intention is from scanning the implementation. (Similarly for the other new classes.)